src/sentry/api/endpoints/organization_environments.py

```diff
@@ -1,21 +1,51 @@
+from typing import TypedDict
+
+from drf_spectacular.utils import extend_schema
 from rest_framework.request import Request
 from rest_framework.response import Response
 
+from sentry.api.api_owners import ApiOwner
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases import OrganizationEndpoint
 from sentry.api.helpers.environments import environment_visibility_filter_options
 from sentry.api.serializers import serialize
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST, RESPONSE_FORBIDDEN
+from sentry.apidocs.examples.environment_examples import EnvironmentExamples
+from sentry.apidocs.parameters import EnvironmentParams, GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.models.environment import Environment, EnvironmentProject
 
 
+class OrganizationEnvironmentResponseType(TypedDict):
+    id: int
+    name: str
+
+
+@extend_schema(tags=["Environments"])
```

Comment:
is it necessary to make a whole new section for this or can we put this under Organizations? 🤔

Reply:
hmm i considered that, but my only concern is as we slowly start publishing more endpoints, if we keep throwing everything org scoped into Organization, it might become more difficult to search for something 🤔